<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="BiswaranjanD-PC" name="Command line test 5c7a6ed2-29ce-482a-9feb-0cf8f15a109b" tests="1" failures="1" timestamp="29 Nov 2017 09:27:25 GMT" time="109.78" errors="0">
  <testcase name="userMangement_MPOINT_B" time="104.04" classname="com.loylty.mobileautomation.testcases.Mpoint_B_Test">
    <failure type="java.lang.AssertionError" message="User management unsucessful expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: User management unsucessful expected [true] but found [false]
	at com.loylty.mobileautomation.testcases.Mpoint_B_Test.userMangement_MPOINT_B(Mpoint_B_Test.java:82)
	at com.loylty.automationframework.TestRunner.main(TestRunner.java:116)
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:294)
	at java.lang.Thread.run(Thread.java:745)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- userMangement_MPOINT_B -->
</testsuite> <!-- Command line test 5c7a6ed2-29ce-482a-9feb-0cf8f15a109b -->
